src/app/aboutUs/aboutUs.module.scss

```diff
@@ -0,0 +1,382 @@
+.aboutUs {
+	display: flex;
+	flex-direction: column;
+	width: 100%;
+	margin: 0 auto;
+	padding: 0 80px 75px;
+	position: relative;
+	overflow: hidden;
+
+	&__rectangle {
+		position: absolute;
+		top: 0;
+		left: 50%;
+		transform: translateX(-50%);
+		height: 96px;
+		width: 80%;
+		background-color: var(--rectangle-about);
+		border-bottom-left-radius: 67px;
+		border-bottom-right-radius: 67px;
+
+		@media (max-width: 1920px) {
+			height: 72px;
+		}
+
+		@media (max-width: 1440px) {
+			height: 54px;
+		}
+
+		@media (max-width: 1170px) {
+			height: 44px;
+		}
+
+		@media (max-width: 1024px) {
+			height: 38px;
+		}
+
+		@media (max-width: 768px) {
+			height: 29px;
+		}
```

Comment:
can you optimize it with clamp css function here and in other similar places as possible?